content/pages/features.html

```diff
@@ -71,9 +71,10 @@ <h3>A Test Plan is a collection of Test Cases</h3>
                     </ul>
                 </div>
 
-                <div style="float: right; clear:right;margin-bottom: 5rem;">
-                    <img src="/images/features/testplan.png" class="margin0 shadow">
+                <div class="testplan">
```

Comment:
There's no definition for the `.testplan` class with the removed properties.

Reply:
What do You mean by that?

The styles for `.testplan` are defined on line `502`, if that's what You're talking about.

Or are You saying that You want the properties `float: right` and `clear: right` to be returned, since they are no longer used?

Reply:
Yes, I'm saying the float and clear properties need to come back. Without them this change breaks the existing layout when on a large screen.

Reply:
Thanks for the clarification, would not have understood it otherwise.